src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -15,7 +19,11 @@ public HashedPassword(final String hashedPassword) {
    * Returns true if the hashed password looks like it could be a hash.
    */
   public boolean isValidSyntax() {
-    return value != null && value.startsWith(MD5Crypt.MAGIC);
+    return value != null && (value.startsWith(MD5Crypt.MAGIC) || value.matches("^\\$2a\\$.{56}$"));
+  }
+
+  public boolean isBcrypted() {
+    return value != null && value.matches("^\\$2a\\$.{56}$");
```

Comment:
Can we extract this test so it is shared with the previous method?  Maybe extracting an `isMd5Crypted()` method and then `isValidSyntax()` can just be `return isMd5Crypted() || isBcrypted();`.